### PR TITLE
adding config for cargo-bitrig-64

### PR DESCRIPTION
**Pull request by dhuseby**
I think this correctly adds the configuration for cargo-bitrig-64.

### CODE REVIEW

**Comment by brson** [comment]
Thanks! I'll deploy this tomorrow.

File: master/master.cfg [hunk 1]
```diff
@@ -191,6 +191,7 @@ nogate_builders = [
     "auto-bitrig-64-opt", 
     "auto-linux-musl-64-opt",
     "auto-win-msvc-64-opt",
+    "cargo-bitrig-64"
```
Comment (alexcrichton) [inline master/master.cfg]: I think that this array is only used for the auto builders themselves, not necessarily the cargo builders unfortunately. I also unfortunately am not familiar with how `nogate_builders` works :(

**Comment by dhuseby** [comment]
Hrm...I looked more closely and it looks like we only gate on builders with names that start with "auto": https://github.com/rust-lang/rust-buildbot/blob/master/master/master.cfg#L1824

The BSKiller class takes a lambda filter that only considers builds with names that start with "auto" and are not in the nogate_builders list.

So my change to the nogate_builders list doesn't do anything.

**Comment by dhuseby** [comment]
I just removed the change to nogate_builders since it doesn't do what I think it does.

**Comment by dhuseby** [comment]
@brson status?

**Comment by brson** [comment]
I'm going to deploy this with some other changes tonight.

**Comment by brson** [comment]
Deployed.

**Pull request merged**